plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -168,8 +166,22 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 				hr = enum_adapter->GetDesc1(&desc);
 				if (S_OK == hr && (GSAdapter(desc) == adapter_id || adapter_id == "default"))
 				{
-					if (desc.VendorId == 0x10DE)
-						nvidia_vendor = true;
+					// D3D GPU vendor check.
+					switch (desc.VendorId)
+					{
+						case 0x10DE:
+							m_gpu_vendor = GPU_VENDOR::VENDOR_NVIDIA;
```

Comment:
if you use an enum class (which I think is good btw) then you have nicer scoping and you don't need something like `GPU_VENDOR::VENDOR_NVIDIA` you can just go `GPU_VENDOR::NVIDIA`.